catalog-rest-service/src/main/java/org/openmetadata/catalog/events/ChangeEventHandler.java

```diff
@@ -42,13 +42,15 @@ public Void process(ContainerRequestContext requestContext, ContainerResponseCon
       ChangeEvent changeEvent = getChangeEvent(method, responseContext);
       if (changeEvent != null) {
         LOG.info("Recording change event {} {}", changeEvent.getDateTime().getTime(), changeEvent);
+        EventPubSub.publish(changeEvent);
         if (changeEvent.getEntity() != null) {
           changeEvent.setEntity(JsonUtils.pojoToJson(changeEvent.getEntity()));
         }
+
         dao.changeEventDAO().insert(JsonUtils.pojoToJson(changeEvent));
       }
     } catch (Exception e) {
-      LOG.error("Failed to capture change event for method {} due to {}", method, e);
+      LOG.error("Failed to capture change event for method {} due to ", method, e);
```

Comment:
this wouldn't log the error right?

Reply:
This would log the entire exception stack trace based on the last exception parameter `e` we are passing. There is no need to add `{}` for an exception. If we do add it, only an exception message is logged.